### ISSUE TITLE
Pin location for Brisbane is actually closer to Mt Isa

### ISSUE THREAD

**Issue by damies13** [linuxmint/mintinstall#265]
It's only slightly out, just a mere 1800km, thought if it were the capital of a state in any other country it probably wouldn't still be in the same state, let alone the same country.

I've attached a screen shot, which i've added an arrow, showing where the pin should have been, also a screen shot of the location of Brisbane and Mt Isa

<img width="981" alt="Screen Shot 2019-11-05 at 8 34 51 pm" src="https://user-images.githubusercontent.com/15232871/68201585-c224fc00-000d-11ea-8a9e-eb1d7612dcc2.png">

<img width="486" alt="Screen Shot 2019-11-05 at 8 48 13 pm" src="https://user-images.githubusercontent.com/15232871/68201593-c5b88300-000d-11ea-9f8a-800aeefdcf6c.png">

This issue is only cosmetic and only affects the installer as far as I know, so not high priority.

**Comment by gm10** [comment]
This should have been submitted under the https://github.com/linuxmint/ubiquity project, but it likely won't matter. 

I looked this up since I was curious. This is intended behaviour of `libtimezonemap`, which the installer uses. The pin represents the first database entry matching the timezone of the location you searched for, and in this case it is indeed Mount Isa. It's the same for any location, the pin only represents the city you searched for if it happens to be the same as the first matching database entry.

PS: Don't get me wrong, I'm not saying this is the most elegant solution, just explaining what is happening.

**Comment by damies13** [comment]
Hi gm10,

Is there a simple way simply move this issue to that project? Or do I just copy and paste the text into a new issue for them?

I guess the way you described it working is probably good enough in tiny European countries, and the tiny little states they have in America. As I said before this is only cosmetic, but it does affect the users first impression of the OS, so it would be good to have it fixed.

**Comment by gm10** [comment]
Yes, it can be moved, but not by you, only by team members (and I've never seen them do that). All you could do is close it here and recreate it over there. I can all but guarantee you that no action will be taken on this though. This is an upstream library, I doubt Clem will fork it just for this. But upstream they are not bothered about addressing this, either, bug reports have been pending for many years.
